search-enterprise/latest/config/transactional.md

```diff
@@ -0,0 +1,382 @@
+* The database results include hidden nodes. You can exclude them in the query. The Elasticsearch index results will never include hidden nodes and respects the index control aspect.
+* The database post filters the results to apply permissions. As a result, no total count can be provided and large result sets are not well supported. This also affects paging behaviour. Permission evaluation is truncated by time or number of evaluations. TMDQ is not intended to scale to more than 10s of thousands of nodes. It will not perform well for users who can read one node in a million. It cannot and will not tell you how many results matched the query. To do this could require an inordinate number of permission checks. It does enough to give you the page requested. The Elasticsearch index can apply permissions at query and facet time to billions of nodes. For the same reason, do not expect any aggregation support in TMDQ.
+* `CONTAINS()` support is complicated. The pure CMIS part of the query and `CONTAINS()` part are melded together into a single abstract query representation. By default, in CMIS the `CONTAINS()` expression implies full text search, so the queries will go to the Elasticsearch index.
+* The database does not score. It will return results in some order that depends on the query plan, unless you ask for specific ordering. A three part `OR` query, where some documents match more than one constraint, is treated as equal. For Elasticsearch index queries, the more parts of an `OR` match, the higher is the score. The docs that match more optional parts of the query will come higher up.
+* Queries from Share will not use TMDQ as they will most likely have a full text part to the query and ask for facets.
+
+### Exact match and patterns
+
+TMDQ can support exact match on all properties (subject to database collation) regardless of the property index configuration in the data model. All text properties can support pattern matching. The database index supports a fixed number of leading characters. The database store a maximum string size before it overflows to another form. Only short form strings can be used in database queries.
+
+Elasticsearch supports exact match on all non-text properties. Text properties only support exact and pattern matches if set to tokenised `both` or `false` in the data model. Elasticsearch provides supports values up to approximately 32,700 UTF-8 bytes.
+
+The following specific CMIS QL fields are supported:
+
+* `cmis:parentId`
+* `cmis:objectcId`
+* `cmis:objectTypeId`
+* `cmis:baseTypeId`
+* `cmis:contentStreamMimeType`
+* `cmis:contentStreamLength`
+
+The following CMIS QL comparison operators are supported:
+
+* `=`, `!=,` `<>`, `<`, `<=`, `>`, `>=`
+* `IN`, `NOT IN`, `LIKE`
+
+The following AFTS exact matches and patterns are supported:
+
+* `=<field>:term`
+* `=<field>:ter*`
+* `=<field>:*erm`
+
+### Full text search for CMIS QL and AFTS
```

Comment:
I'm not sure I understand what this section means. It doesn't mention TMDQ and the contents of the section don't mention full text search (apart from the acronym AFTS).

Reply:
Ive removed it,